server/views/partials/challenge-modals.jade

```diff
@@ -19,3 +19,16 @@
                 h3 This will restore your code editor to its original state.
                 a.btn.btn-lg.btn-warning.btn-block#reset-button(href='#', data-dismiss='modal', aria-hidden='true') Clear my code
                 a.btn.btn-lg.btn-primary.btn-block(href='#', data-dismiss='modal', aria-hidden='true') Cancel
+
+#ask-for-help-modal.modal(tabindex='-1')
+    .modal-dialog.animated.fadeInUp.fast-animation
+        .modal-content
+            .modal-header.challenge-list-header Ask for help?
+                a.close.closing-x(href='#', data-dismiss='modal', aria-hidden='true') ×
+            .modal-body
+                h3
+                  | If you've already tried the&nbsp;
+                  a(href='https://forum.freecodecamp.org/t/the-read-search-ask-methodology-for-getting-unstuck/137307', target='_blank') Read-Search-Ask&nbsp;
+                  | Method, then you can ask for help on the freeCodeCamp forum.
+                a.btn.btn-lg.btn-primary.btn-block#ask-for-help-button(name='_csrf', value=_csrf) Create a help post on the forum
```

Comment:
@piotrek1543 I kept it as descibed in the issue by @QuincyLarson .
However, if you want I will make it short.